docs/topics/api/addons.rst

```diff
@@ -44,6 +44,10 @@ This endpoint allows you to search through public add-ons.
     :query string appversion: Filter by application version compatibility. Pass the full version as a string, e.g. ``46.0``. Only valid when the ``app`` parameter is also present.
     :query string author: Filter by exact author username. Multiple author names can be specified, separated by comma(s), in which case add-ons with at least one matching author are returned.
     :query string category: Filter by :ref:`category slug <category-list>`. ``app`` and ``type`` parameters need to be set, otherwise this parameter is ignored.
+    :query boolean featured: Filter to only featured add-ons.
+        If ``app`` is provided as a parameter then only featured collections targetted to that application are taken into account.
+        If ``lang`` is provided then only featured collections for that language, or no language, are taken into account. Both ``app`` and ``lang`` can be provided to filter to addons that are featured in that application and for that language, or no language.
```

Comment:
I find "or no language" highly confusing, are featured collections with no language simply targeting **all** languages? If so, we should document it like that, not yet sure about the phrasing though.

Reply:
I'm confused what you want documenting 😆 
Yes, most featured collections don't target any locale - that's the norm.  _Some_ target a particular locale, and it's those ones that get filtered in/out if you request a lang.